src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/ReportedAdjudicationController.kt

```diff
@@ -243,4 +254,19 @@ class ReportedAdjudicationController {
 
     return ReportedAdjudicationResponse(reportedAdjudication)
   }
+
+  @PreAuthorize("hasRole('ADJUDICATIONS_REVIEWER')")
```

Comment:
As this endpoint creates a record in the database we could also enforce `hasAuthority('SCOPE_write')` here?